map/framework.cpp

```diff
@@ -2229,7 +2231,12 @@ place_page::Info Framework::BuildPlacePageInfo(place_page::BuildInfo const & bui
 
   // @TODO: (KK) Enable track selection.
   // The isTrackSelectionEnabled should be removed to enable the track selection when the UI will be implemented.
-  const bool isTrackSelectionEnabled = false;
+  #if defined(__APPLE__)
```

Comment:
As there's no track selection in the Qt mac app, it should be TARGET_OS_IPHONE instead of __APPLE__.

You may need to include "std/target_os.hpp"

Reply:
Thanks!